src/Spinner/Spinner.js

```diff
@@ -248,11 +249,16 @@ module.exports = kind(
 	*/
 	accessibilityLive: 'off',
 
-	/**
-	* @default Loading
-	* @type {String}
-	* @translated loading message
-	* @public
-	*/
+	
 	loadingMessage: $L('Loading')
 });
+
+/**
+* @type {String}
+* @translated loading message
+* @public
+* @usage in application
+* @var Spinner = require('moonstone/Spinner');
+* @this.$.spinner.set('accessibilityLabel', Spinner.LOADING_STRING);
+*/
+module.exports.LOADING_STRING = loadingString;
```

Comment:
I think you should use markdown syntax for the sample code and please reference below comment.
```
/**
* Translated loading message.
*
* Usage in application.
* ```
* var Spinner = require('moonstone/Spinner');
* this.$.spinner.set('accessibilityLabel', Spinner.LOADING_STRING);
* ```
*
* @type {String}
* @public
*/
```